### DISCUSSION TITLE
Question: why division into modules Admin & Auxiliary?

### DISCUSSION THREAD

**Discussion by eydun** [nette-examples/user-authentication#7] [General]
Why is the ErrorPresenter placed into it's own Auxiliary-module? (just trying to understand)

**Comment by dg** [comment]
It looks strange because there is no other module, like Front.

**Comment by eydun** [comment]
Thanks. I have not encountered the name "auxiliary" in this context before. Maybe the word "helpers" would also work?

**Reply by dg** [reply]
Sure, it could have been called Helpers.
